verticapy/tests/vDataFrame/test_vDF_plot_plotly.py

```diff
@@ -0,0 +1,76 @@
+"""
+(c)  Copyright  [2018-2023]  OpenText  or one of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+
+# Pytest
+import pytest
+
+# Standard Python Modules
+import datetime, os, sys
+
+# Other Modules
+import plotly.express as px
+import plotly
+
+# VerticaPy
+import verticapy
+import verticapy._config.config as conf
+from verticapy import drop, set_option
```

Comment:
why loading so many things if you are just using few. Import only what is needed.

Reply:
done

Reply:
you're importing config as conf. Use:
 - conf.set_option
No need to import set_option.